src/test/java/com/dabsquared/gitlabjenkins/publisher/GitLabAcceptMergeRequestPublisherTest.java

```diff
@@ -66,7 +66,7 @@ public void success() throws IOException, InterruptedException, GitLabApiExcepti
         publish(mockSimpleBuild(GITLAB_CONNECTION_V4, Result.SUCCESS));
 
         mockServerClient.verify(
-            prepareAcceptMergeRequestWithSuccessResponse("V4", MERGE_REQUEST_IID, null));
+            prepareAcceptMergeRequestWithSuccessResponse("v4", MERGE_REQUEST_IID, null));
```

Comment:
Should be "V4" to be consistent?

Reply:
Nope! GitLab uses v4 in their URLs but GitLab4J-API uses V4 in ApiVerison Enums. Inconsistency problems!